### ISSUE TITLE
std.algorithm

### ISSUE THREAD

**Issue by joelcnz** [adamdruppe/arsd#82]
With DMD 2.071.0 I now get:
../arsdLib/source/arsd/dom.d(4811,17): Deprecation: package std.algorithm is not accessible here, perhaps add 'static import std.algorithm;'

**Comment by adamdruppe** [comment]
On Wed, Apr 06, 2016 at 08:04:47PM -0700, Joel Christensen wrote:
> With DMD 2.071.0 I now get:

Give the new version a try now, it all works on my box for Windows
and Linux now.

**Comment by joelcnz** [comment]
I didn't get the deprecation error, but got this instead (a list of them):

../BibleLib/source/bible/base.d(328,8): Error: no property 'n' for type 'arsd.dom.Element'

**Comment by adamdruppe** [comment]
On Wed, Apr 06, 2016 at 09:33:11PM -0700, Joel Christensen wrote:
> ../BibleLib/source/bible/base.d(328,8): Error: no property 'n' for type 'arsd.dom.Element'

Is that an attribute on the element?

I used to put attributes on a naked opDispatch, but that actually caused
quite a few problems, so I deprecated that a while ago and removed it
somewhat recently.

Instead, try `element.attrs.n` - the `.attrs` medium gives you access without
the problems the naked one had.

(I still allow convenient attribute fetching for very common ones like id,
href, and a few others, but for less common or non-standard ones, use
.attrs.)
